templates/examples/qr-code-generator/package.json

```
@@ -0,0 +1,17 @@
+{
+	"name": "qr-code-generator",
+	"version": "0.0.0",
+	"private": true,
+	"scripts": {
+		"deploy": "wrangler publish",
```

```suggestion
		"deploy": "wrangler deploy",
```

As per https://github.com/cloudflare/workers-sdk/pull/3486, it'd make sense to align this with the rest of the templates using `deploy` over `publish` now.

---

Thanks @Cherry!